### ISSUE TITLE
Unable to turn off verbose when running tests

### ISSUE THREAD

**Issue by kevchandler** [go-lang-plugin-org/go-lang-idea-plugin#2061]
This may be by design but there doesn't appear to be a way to run tests in silent mode, i.e. not logging to the console if tests pass.

When I attempt to turn verbose mode to false by adding the Program argument, <code>-test.v=false</code> then I get the error, <code>go test: v flag may be set only once</code>

I would like to be able to control the level of output generated whilst running tests myself

I'm using Intellij IDEA 14.1.5 and the plugin from https://plugins.jetbrains.com/plugins/alpha/5047

**Comment by zolotov** [comment]
> This may be by design

Of course it is by design, IDEA can't parse tests output if there is no one.

Won't fix.
